src/MainWindow.vala

```diff
@@ -271,6 +274,46 @@ public class Tasks.MainWindow : Hdy.ApplicationWindow {
         });
     }
 
+    private void dialog_refresh_collection_error (Error e) {
+        string error_message = e.message;
+
+        GLib.Idle.add (() => {
+            var error_dialog = new Granite.MessageDialog (
+                _("Refresh collection failed"),
+                _("The backend may be unavailable or unable to read from."),
```

Comment:
Is there a more user-facing way to phrase this? Maybe something like:
```
Unable to get updates from "me@example.com"
The tasks stored there may be unavailable or unable to be read from.
```

Reply:
Done.